frontend/app/src/components/price-manager/latest/LatestPriceTable.vue

```diff
@@ -204,17 +203,10 @@ const filteredPrices = computed(() => {
   const data = [...get(latestPrices)].filter(({ fromAsset }) => {
     return !isNft(fromAsset) && (!filter || fromAsset === filter);
   });
-  return data.map(item => {
-    const assetPrice =
-      item.toAsset === CURRENCY_USD ? One : getAssetPrice(item.toAsset);
-
-    return {
-      ...item,
-      usdPrice: (assetPrice
-        ? assetPrice.multipliedBy(item.price)
-        : NoPrice) as BigNumber
-    };
-  });
+  return data.map(item => ({
+    ...item,
+    usdPrice: getAssetPrice(item.fromAsset)
```

Comment:
Is there any reason why the `item.toAsset` now seems to be `item.fromAsset`? was this a bug?

Reply:
ok I will check it thanks.

Reply:
Keep in mind that the prices still wrong, @prettyirrelevant still fixing the prices